templates/permafrost/huc.html

```diff
@@ -0,0 +1,352 @@
+{% extends 'base.html' %}
```

Comment:
We may want to think about how much of these previews we include, probably only necessary to show some of the sample response and make it easier to scroll to descriptions below it. Fine for now.